pymc4/distributions/mixture.py

```diff
@@ -19,11 +26,31 @@ class Mixture(Distribution):
     ----------
     p : array of floats
         p >= 0 and p <= 1
-        the mixture weights, in the form of probabilities
+        the mixture weights, in the form of probabilities, must sum to one.
```

Comment:
A key thing that we need to document here is that `p` has to sum 1 over a very specific axis. We need to specify which axis has to sum to one (I don't remember exactly but I think that tfp uses the last one). Also, we could change the type of `p` to something more appropriate, like a `tf.Tensor` and add a type hint for mypy maybe.

Reply:
Right, I added type hints for the probabilities as well as for the distribution. Should I unify this for the other distributions as well (I mean types should be tensors instead of `np.array`s or so).